### PR TITLE
Update job and job request templates

### PR DESCRIPTION
**Pull request by tomodwyer**
- Update:
  - font weights
  - Job detail template
  - Job request template
- Remove unused `convert-csv-to-json` package

### CODE REVIEW

**Comment by ghickman** [comment]
`.prose-width` on the lead text for a Job makes it wrap in an awkward place, do we need to restrict that block of text?

File: jobserver/templates/job_request_detail.html [hunk 1]
```diff
@@ -10,227 +11,290 @@
...
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              {% for job in jobrequest.jobs.all %}
+              <tr>
+                <td>{{ job.identifier }}</td>
+                <td>{{ job.status }}</td>
+                <td>{{ job.action }}</td>
+                <td><a class="btn btn-link p-0" href="{{ job.get_absolute_url }}">View</a></td>
```
Comment (ghickman) [inline jobserver/templates/job_request_detail.html:165]: Should we ditch the `View` here and just link the identifier instead?

**Review by ghickman — APPROVED** [review]
Looks great, couple of aesthetic points.

**Comment by tomodwyer** [comment]
@ghickman agreed on both 👍

**Pull request merged**